packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.md

```diff
@@ -1,25 +1,6 @@
 # component-description
 
-Selector input for chosing quantity of products
+Component with input to choose numeric values and describe quantity.
 
-# common-usage
-
-<br>
-<SfQuantitySelector :stock="2"  :qty="1" />
-
-```html
-
-<template>
-  <SfQuantitySelector :stock="stock" :qty="qty" />
-</template>
-
-<script>
-  import { SfQuantitySelector } from "@storefront-ui/vue";
-
-  export default {
-    components: {
-      SfQuantitySelector
-    }
-  };
-</script>
-```
+# storybook-iframe-height
+5rem
```

Comment:
If you change the MD to the new docs format, this whole PR will depend on #480.
imo leave the format as-is for now, so we can merge this PR to develop. Then, edit the MD file inside #502. #502 will be merged together with #480, so then it will be fine.

Reply:
Ok, I'll keep it the old way then :)